### ISSUE TITLE
No number of days in my sessions in community 

### ISSUE THREAD

**Issue by cemsbr** [tslocum/MeditationAssistant#8]
I can see other people's number of consecutive days in their session entries, but not mine. The same happens with my friend and we cannot see each other's numbers.

**Comment by tslocum** [comment]
Could you please clarify where you mean?  Are you referring to the community-wide display, with everyone's streaks?  However, on your device, a streak is displayed.  Am I understanding you correctly?

**Comment by cemsbr** [comment]
Yes, I'm referring to the community-wide display. I can see the streak of most people (e.g. "3rd day of meditation"), but some, like me and my friend, have no streak message. How can I include my streak?

**Comment by cemsbr** [comment]
I realized that the streak info is only available if you use the "Publish" button after your meditation session. However, if you upload all your sessions at once, no streak is displayed in the community display.
